Source/FieldSolver/London/London.H

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2020 Revathi Jambunathan
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+
+#ifndef LONDON_H
+#define LONDON_H
+
+#include <AMReX_RealVect.H>
+#include <AMReX_REAL.H>
+#include <AMReX_GpuQualifiers.H>
+#include <AMReX_Geometry.H>
+#include <AMReX_Array4.H>
+#include <AMReX_Array.H>
+#include <AMReX_MultiFab.H>
+#include <AMReX_Parser.H>
+
+using namespace amrex;
```

Comment:
Don't add `using namespace` in unscoped sections of header-files.
Otherwise you will "pollute" the namespace scopes of every file that includes this header:
```suggestion
```